### ISSUE TITLE
Order Placement - Suppress numberOfAllocations

### ISSUE THREAD

**Issue by MichaelStaehli** [OpenWealth/OpenWealth#26]
**Requestor:**
Fabian Rutishauser - ZKB

**Submission date:**
20.12.2022

**Priority Requestor:**
Medium

**Effort T-Shirt Size:**
S

**Description:**
Why is this number needed? Suggest to remove it as also the quantity attribute is a way to do a “check sum” logic.

https://github.com/OpenWealth/OpenWealth/blob/9cd160507302454583a74e68fd18e84e68f2bb3e/OrderPlacement/Specs/Resources/bulkOrderDetails.yaml#L22

**Comment by andiahlm** [comment]
@TLusti can we remove the property? What was the background to add it?

**Comment by TLusti** [comment]
> [@TLusti](https://github.com/TLusti) can we remove the property? What was the background to add it?

@andiahlm Yes, this property can be removed, as it is redundant information. Initially was thought to have a check sum. But I would suggest that payloads (and the allocation arrays within) should be sufficient.
